contracts/bond/BondMediator.sol

```diff
@@ -79,6 +85,20 @@ contract BondMediator is BondAccessControl, UUPSUpgradeable, Version {
         return bond;
     }
 
+    /**
+     * @notice Pauses most side affecting functions.
+     */
+    function pause() external whenNotPaused onlyRole(Roles.BOND_ADMIN) {
```

Comment:
Do we want this in BondManager and BondMediator seperately or abstract contract with this common functionality? @CjHare

Reply:
Good question.

The upgradability, pausability and access control are common across all. Having a single abstract parent contract would maximize code reuse ...what are down sides tho 🤔

Reply:
Feel like it limits the roles access control more?

Reply:
> @CjHare (do I need to tag you explicitly for visibility or not?)

No, I get an email/notification on response to any of my comments 😁

Reply:
>Feel like it limits the roles access control more? Like to solely BOND_ADMIN which might make it less useful to be abstracted when an actor other than the BOND_ADMIN needs to pause something.

True, the function modifiers are the gotcha 🤔 
_In some cases pausing may be within the power of `BOND_OPERATORS`_

Reply:
Summary: keep 'em seperate for now 👍